server/src/binary/handlers/consumer_offsets/store_consumer_offset_handler.rs

```diff
@@ -20,10 +21,13 @@ pub async fn handle(
             command.consumer,
             &command.stream_id,
             &command.topic_id,
-            command.partition_id,
-            command.offset,
+            command.partition_id.clone(),
+            command.offset.clone(),
         )
-        .await?;
+        .await
+        .with_error(|_| format!("CONSUMER_OFFSET_HANDLER - failed to store consumer offset for stream_id: {}, topic_id: {}, partition_id: {:?}, offset: {}, session: {}",
```

Comment:
since this with_error method is lazily called only on error, maybe we could try moving the `command.partition_id.clone()`  and offset inside of there.